application/src/main/java/org/togetherjava/tjbot/commands/help/ImplicitAskListener.java

```diff
@@ -114,13 +114,15 @@ private boolean handleIsNotOnCooldown(Message message) {
         String threadDescription = lastHelpThread == null ? "your previously created help thread"
                 : lastHelpThread.getAsMention();
 
-        message.getChannel()
-            .sendMessage("""
-                    %s Please use %s to follow up on your question, \
-                    or use `/ask` to ask a new questions, thanks."""
-                .formatted(author.getAsMention(), threadDescription))
-            .flatMap(any -> message.delete())
+        MessageUtils.mentionSlashCommand(message.getGuild(), AskCommand.COMMAND_NAME)
+            .map(command -> message.getChannel()
+                .sendMessage("""
+                        %s Please use %s to follow up on your question, \
+                        or use %s to ask a new questions, thanks."""
+                    .formatted(author.getAsMention(), threadDescription, command))
+                .flatMap(any -> message.delete()))
             .queue();
```

Comment:
Use flatmap instead of a map, with a flatmap inside the map
```suggestion
            .flatMap(command -> message.getChannel()
                .sendMessage("""
                        %s Please use %s to follow up on your question, \
                        or use %s to ask a new questions, thanks."""
                    .formatted(author.getAsMention(), threadDescription, command)))
             .flatMap(any -> message.delete()))
            .queue();
```